codec/pom.xml

```diff
@@ -119,6 +119,12 @@
       <optional>true</optional>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-compress</artifactId>
+      <optional>true</optional>
+    </dependency>
```

Comment:
I would prefer to keep this as test dependency only

Reply:
Done